README.md

```diff
@@ -1,2 +1,48 @@
 # nazareth-events
-A web app for viewing events in Nazareth
+A web app for viewing events in Nazareth.
```

Comment:
Write that it's built on top of the Nazareth Open Tourism Platform (maybe with a link)